src/TinyLittleMvvm/IWindowManager.cs

```diff
@@ -1,11 +1,13 @@
 ﻿using System.Windows;
 using Microsoft.Extensions.DependencyInjection;
 
-namespace TinyLittleMvvm {
+namespace TinyLittleMvvm
+{
     /// <summary>
     /// Declares methods to show windows.
     /// </summary>
-    public interface IWindowManager {
```

Comment:
Please pay attention to coding style, see comment above regarding placement of opening braces.